content/docs/lifting-state-up.md

```diff
@@ -299,32 +299,31 @@ class Calculator extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/WZpxpz?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/WZpxpz?editors=0010)
 
-Now, no matter which input you edit, `this.state.temperature` and `this.state.scale` in the `Calculator` get updated. One of the inputs gets the value as is, so any user input is preserved, and the other input value is always recalculated based on it.
+Այժմ, կարևոր չէ, թե որ input-ն եք դուք խմբագրում, `Calculator`-ում `this.state.temperature`-ը և `this.state.scale`-ը կմնան թարմացված։ input-ներից մեկը ստանում է արժեքը այնպես ինչպես կա, մյուսը input-ի արժեքը միշտ վերահաշվարկվում է հիմնվելով դրա վրա։
```

Comment:
"so any user input is preserved" չես թարգմանել․․ "այսպիսով օգտագործողի ցանկացած input պահպանվում է"